### ISSUE TITLE
Notices and fatals with Jetpack 3.7.0-dev on fresh install

### ISSUE THREAD

**Issue by natebot** [Automattic/vip-quickstart#455]
I just did a fresh install of vip-quickstart and I got php notices and fatals due to Jetpack 3.7.0-dev
After following the set up steps, visiting http://vip.local/ will display these notices .

```
Notice: Use of undefined constant AUTH_COOKIE - assumed 'AUTH_COOKIE' in /srv/www/wp/wp-includes/pluggable.php on line 801

Notice: Use of undefined constant LOGGED_IN_COOKIE - assumed 'LOGGED_IN_COOKIE' in /srv/www/wp/wp-includes/user-functions.php on line 246
```
There's also an unlogged fatal (white screen) at `vip.local/wp-admin/`
I also get the same PHP notices when using WP-CLI in the guest machine.
The stack trace suggests an issue with Jetpack:

```
HP Notice:  Use of undefined constant LOGGED_IN_COOKIE - assumed 'LOGGED_IN_COOKIE' in /srv/www/wp/wp-includes/user-functions.php on line 246
PHP Stack trace:
PHP   1. {main}() /usr/local/src/wp-cli/bin/wp:0
PHP   2. include() /usr/local/src/wp-cli/bin/wp:4
PHP   3. include() phar:///usr/local/src/wp-cli/bin/wp/php/boot-phar.php:5
PHP   4. require() phar:///usr/local/src/wp-cli/bin/wp/php/wp-cli.php:26
PHP   5. include_once() phar:///usr/local/src/wp-cli/bin/wp/php/wp-settings-cli.php:204
PHP   6. require_once() /srv/www/wp-content/plugins/jetpack/jetpack.php:50
PHP   7. Jetpack_Autoupdate::init() /srv/www/wp-content/plugins/jetpack/class.jetpack-autoupdate.php:282
PHP   8. Jetpack_Autoupdate->__construct() /srv/www/wp-content/plugins/jetpack/class.jetpack-autoupdate.php:28
PHP   9. get_currentuserinfo() /srv/www/wp-content/plugins/jetpack/class.jetpack-autoupdate.php:35
PHP  10. apply_filters() /srv/www/wp/wp-includes/pluggable.php:119
PHP  11. call_user_func_array() /srv/www/wp/wp-includes/plugin.php:213
PHP  12. wp_validate_logged_in_cookie() /srv/www/wp/wp-includes/plugin.php:213
```

I have disabled Jetpack with WP-CLI and the the notifications and fatal have stopped. I replaced Jetpack 3.7.0-dev with Jetpack 3.6.1 and the fatals and notices stopped.

**Comment by westonruter** [comment]
I noticed the same. It seems specific to the `3.7.0-beta3` tag, with the issue being fixed in `3.7.0-beta4`. _However_, on `3.7.0-beta4` I then had a separate error where all admin access was blocked in a multisite network. So I had to revert to Jetpack 3.6.1 as well.

**Comment by westonruter** [comment]
For the undefined constant issue, see https://github.com/Automattic/jetpack/issues/2640
